src/client/wallet/Transfer.less

```diff
@@ -54,6 +54,14 @@
         .ant-input-group-addon {
           text-align: right;
           width: 160px !important;
+          position: relative;
+        }
+
+        .ant-radio-group {
+          position: relative;
+          top: 50%;
+          -webkit-transform: translateY(-50%);
```

Comment:
We have autoprefixer that adds browser-specific styles automatically.

Reply:
This is really cool. Can you please point me where/how you do that? Browser prefix are really painful, i have VSCode plugin to take care of that now, but would be great to automate it.